programs/clearing_house/src/math/fulfillment.rs

```diff
@@ -10,6 +10,11 @@ pub fn determine_fulfillment_methods(
 ) -> ClearingHouseResult<Vec<FulfillmentMethod>> {
     let mut fulfillment_methods = vec![];
 
+    // todo:
```

Comment:
can prob just delete this

Reply:
@0xbigz wys

Reply:
yea we can put in the notion doc instead